torch/_dynamo/convert_frame.py

```diff
@@ -690,9 +691,19 @@ def compile_inner(
         hooks: Hooks,
         transform: Callable[[List[Instruction], Dict[str, Any]], Any],
     ) -> Optional[GuardedCode]:
-        with dynamo_timed("_compile.compile_inner", phase_name="entire_frame_compile"):
-            with CompileTimeInstructionCounter.record():
-                return _compile_inner(code, one_graph, hooks, transform)
+        with contextlib.ExitStack() as stack:
```

Comment:
Am I understanding correctly that this is just a neat way to add
```
with _WaitCounter("pytorch.wait_counter.dynamo_compile").guard()
```
to the previous nested `with` block?

Reply:
yes